radixdlt/src/test/java/com/radixdlt/network/messaging/MessageDispatcherTest.java

```diff
@@ -63,12 +67,32 @@
 		DummySignedMessage() {
 			super(0);
 		}
+
+		@Override
+		public boolean equals(Object o) {
+			return o == null || getClass() != o.getClass();
```

Comment:
This seems wrong, but not sure it matters here.  Perhaps just using default identity hash and equals would be fine, and just removing these methods?

Reply:
Probably should be 
```
return o != null && getClass() == o.getClass();
```
if we want them all to be equal.